lstchain/calib/camera/r0.py

```diff
@@ -84,6 +86,17 @@ class LSTR0Corrections(CameraR0Calibrator):
                             allow_none=True,
                             help='Path to the LST pedestal binary file'
                             ).tag(config=True)
+    calibration_path = Unicode(
+        '',
+        allow_none=True,
+        help='Path to LST calibration file'
+    ).tag(config=True)
+
+    allowed_tels = List(
```

Comment:
Is this `allowed_tels` for the EventSource? If yes, this should be an alias to that config, not it's own option